EcpBtBot/DialogFlow/DialogFlowResponse.py

```diff
@@ -31,14 +31,17 @@ def fulfillment(self):
 
 class Metadata:
     def __init__(self,metadata):
-        self.intentId=metadata['intentId']
-        self.webhookUsed=metadata['webhookUsed']
-        self.webhookForSlotFillingUsed=metadata['webhookForSlotFillingUsed']
-        self.intentName=metadata['intentName']
+        self.intentName=""
+        if 'intentName' in metadata.keys():
+            self.intentName=metadata['intentName']
 class Fulfillment:
     def __init__(self,fulfillment):
         self.speech=fulfillment['speech']
         self.messages=fulfillment['messages']
+        self.quickreplies=[]
+        for QuickReply in self.messages:
```

Comment:
do not start variable name with a capital letter.

Reply:
ok